doc/user-guide/Push-notifications.md

```diff
@@ -171,7 +173,7 @@ After this step, you need to have the `pubsub` host (here `pubsub.mypubsub.com`)
 
 ### Enabling push notifications
 
-The next and the last step is to enable push notifications on the server that handles your messages (and have [mod_push][] enabled).
+The next and the last step is to enable push notifications on the server that handles your messages (and have [mod_event_pusher_push][] enabled).
```

Comment:
The next and the last step is to enable push notifications on the server that handles your messages (and has [mod_event_pusher_push][] enabled). ?